cmd/cql/internal/create.go

```diff
@@ -18,38 +18,49 @@ package internal
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
+	"math"
+	"os"
+	"strings"
+	"time"
+
+	"gopkg.in/cheggaaa/pb.v1"
 
 	"github.com/CovenantSQL/CovenantSQL/client"
+	"github.com/CovenantSQL/CovenantSQL/crypto/hash"
+	"github.com/CovenantSQL/CovenantSQL/proto"
 )
 
+var meta client.ResourceMeta
+
 // CmdCreate is cql create command entity.
 var CmdCreate = &Command{
-	UsageLine: "cql create [common params] [-wait-tx-confirm] db_meta_json",
+	UsageLine: "cql create [common params] [-wait-tx-confirm] [db_meta params]",
 	Short:     "create a database",
 	Long: `
 Create creates a CovenantSQL database by database meta info JSON string. The meta info must include
 node count.
 e.g.
-    cql create '{"node": 2}'
-
-A complete introduction of db_meta_json fields：
-
-    target-miners          []string // List of target miner addresses
-    node                   int      // Target node number
-    space                  int      // Minimum disk space requirement, 0 for none
-    memory                 int      // Minimum memory requirement, 0 for none
-    load-avg-per-cpu       float    // Minimum idle CPU requirement, 0 for none
-    encrypt-key            string   // Encryption key for persistence data
-    eventual-consistency   bool     // Use eventual consistency to sync among miner nodes
-    consistency-level      float    // Consistency level, node*consistency_level is the node number to perform strong consistency
-    isolation-level        int      // Isolation level in a single node
+    cql create  -node 2
+
+A complete introduction of db_meta params fields：
+
+    target-miners          []string  // List of target miner addresses
+    node                   uint      // Target node number
+    space                  uint      // Minimum disk space requirement, 0 for none
+    memory                 uint      // Minimum memory requirement, 0 for none
+    load-avg-per-cpu       float     // Minimum idle CPU requirement, 0 for none
+    encrypt-key            string    // Encryption key for persistence data
+    eventual-consistency   bool      // Use eventual consistency to sync among miner nodes
+    consistency-level      float     // Consistency level, node*consistency_level is the node number to perform strong consistency
+    isolation-level        int       // Isolation level in a single node
+    gas-price              uint      // customized gas price
+    advance-payment        uint      // customized advance payment
```

Comment:
Update usage message (the argument is not a 'meta info JSON string' now). Remove these fields introduction cause they will appear in the flags usage message.